tests/Makefile

```
@@ -38,7 +38,10 @@ OPTIONS_LINK=-fexceptions -Wl,-z,max-page-size=16 -Wl,-z,common-page-size=16 -no
 INCLUDES = -I../$(ARCH_LLK_ROOT)/llk_lib -I../$(ARCH_LLK_ROOT)/common/inc -I../$(ARCH_LLK_ROOT)/common/inc/sfpu -I../$(ARCH_LLK_ROOT)/hw_specific/inc
 INCLUDES += -Ifirmware/riscv/common -Ifirmware/riscv/$(CHIP_ARCH)/ -Isfpi/include -Ihelpers/include
 
-FORMAT_ARG := -D$(unpack_A_src) -D$(unpack_A_dst) -D$(unpack_B_src) -D$(unpack_B_dst) -D$(fpu) -D$(pack_src) -D$(pack_dst)
+FORMAT_ARG := -D$(unpack_A_src) -D$(pack_dst)
+ifdef fpu
+	FORMAT_ARG += -D$(unpack_A_dst) -D$(unpack_B_src) -D$(unpack_B_dst) -D$(fpu) -D$(pack_src) 
```

```suggestion
	FORMAT_ARG += $(foreach var, unpack_A_dst unpack_B_src unpack_B_dst fpu pack_src, -D$($(var)))
endif
```
This is a nit. `$(foreach var, list, text)` iterates over each item in a list, makes it easier for later extension.

---

Works like a charm 👍